controllers/cloud.redhat.com/providers/inmemorydb/elasticache.go

```diff
@@ -50,7 +55,7 @@ func (e *elasticache) Provide(app *crd.ClowdApp, config *config.AppConfig) error
 	if !found {
 		return &errors.MissingDependencies{
 			MissingDeps: map[string][]string{
-				"in-memory-db-secret": {app.Name},
+				"secret": {fmt.Sprintf("name: %s, namespace: %s", secretName, app.Namespace)},
```

Comment:
Calling this just `secret` is too generic - we could have multiple "secrets" that are missing dependencies, we either need a more intelligent missing dependency map where we can "add" items to a category, or keep it with a more unique name.

Reply:
Ok.. I like your first idea, but sounds like something we probably need a bit more design around ... just reverted the name for now.